packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
This and license package was probably picked up as changed because of package/dep bumps 🤔